packages/language-service/package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "@types/mdast": "^4.0.0",
     "@types/unist": "^3.0.0",
-    "@volar/language-core": "~1.10.0",
+    "@volar/language-core": "~1.11.1",
```

Comment:
same

---

I'll defer to @remcohaszing on this, elsewhere in unified, we don't update ranges unless the new version is outside the allowed range.
These appear to already be within allowed range.